mdata/store_cassandra_test.go

```diff
@@ -0,0 +1,45 @@
+package mdata
+
+import (
+	"reflect"
+	"testing"
+)
+
+type testCase struct {
+	ttls         []uint32
+	windowFactor int
+	nameFormat   string
+	expected     ttlTables
+}
+
+const oneDay = 24 * 60 * 60
+const oneMonth = oneDay * 30
+const oneYear = oneMonth * 12
+
+func TestGetTTLTables(t *testing.T) {
+	tc := testCase{
+		ttls:         []uint32{oneDay, oneMonth, oneYear},
+		windowFactor: 20,
+		nameFormat:   "metric_%d",
+		expected:     make(ttlTables),
+	}
```

Comment:
ok, improved that and also added tests for a different window factor since that's now easier with the separate test cases